packages/reports/tests/integration/components/navi-request-preview-test.js

```diff
@@ -12,6 +12,7 @@ import hbs from 'htmlbars-inline-precompile';
 let Store, MetadataService, AdClicks, Revenue, Age, UpdateReportAction;
 const textContentArray = selector => findAll(selector).map(el => el.textContent.trim());
 
+// TODO: Delete this
```

Comment:
Any reason this needs to stay for now?

Reply:
Not particularly, I was thinking just delete it in the next PR

Reply:
If we're skipping these tests anyway, might as well just delete the file.